src/js/claims-status/actions/index.jsx

```diff
@@ -84,16 +86,18 @@ export function fetchAppealsSuccess(response) {
   };
 }
 
-// To test this functionality, go to http://localhost:3001/track-claims/appeals-v2/7387389/status
 export function getAppealsV2() {
   return (dispatch) => {
     dispatch({ type: FETCH_APPEALS_PENDING });
-
-    // Fake the fetch by just returning a resolved promice with the object shape we expect
-    //  to get from the api.
-    return setTimeout(() => Promise.resolve(mockData)
-      .then((response) => dispatch(fetchAppealsSuccess(response)))
-      .catch(() => dispatch({ type: SET_APPEALS_UNAVAILABLE })), 4000);
+    return apiRequest(
+      '/appeals_v2',
+      null,
+      (appeals) => dispatch(fetchAppealsSuccess(appeals)),
+      (error) => {
+        Raven.captureException(`vets_appeals_v2_err_get_appeals ${error.message}`);
```

Comment:
Should this be `Raven.captureMessage()`?

Reply:
🤷‍♂️ wasn't sure what our standard usage is across the site; seems like we use both. I think you're right that captureMessage is probably more appropriate though, so updating.